snuba/admin/static/capacity_management/types.tsx

```diff
@@ -0,0 +1,44 @@
+import { ReactNode } from "react";
+
+type AllocationPolicy = {
+  storage_name: string;
+  allocation_policy: string;
+};
+
+type AllocationPolicyConfig = {
+  key: string;
+  value: string;
+  description: string;
+  type: string;
+  params: object;
+};
+
+type AllocationPolicyConfigParams = {
+  name: string;
+  type: string;
+};
+
+type AllocationPolicyParametrizedConfigDefinition = {
+  name: string;
+  type: string;
+  default: string;
+  description: string;
+  params: AllocationPolicyConfigParams[];
+};
+
+type RowData = [
+  ReactNode,
+  ReactNode,
+  ReactNode,
+  ReactNode,
+  ReactNode,
+  ReactNode
+];
```

Comment:
this is a strange type. What's the story behind it?

Reply:
Copied it from runtime config code and added an extra `ReactNode` to work with our use case lol, it's just what the `Table` component requires:

https://github.com/getsentry/snuba/blob/21864c360af65cb9aac0b10866bdab1a3f49f53e/snuba/admin/static/runtime_config/types.tsx#L16